### DISCUSSION TITLE
It is not possible to customize the Queue-view

### DISCUSSION THREAD

**Discussion by Heathcliff77** [alphapapa/listen.el#28] [Ideas]
As far as I can tell it is not possible to customize the Queue-view to your own liking. 
I looked at `listen-queue.el` (I am not an elisp-programmer) and it looks hardcoded.

I am mainly thinking about what columns should be displayed and their order.
Just as an example: I would personally want: `# (playlist track number)` | `Filename (without path) or Title (if present in metadata, it may be missing)` | `Duration`

i.e. 3 columns, that is all I want.

( Side note: The README states that `listen.el` aims to "_be easily extended and customized_" . So I guess developers agree on that this should be possible (in some coming version of listen.el) )

**Comment by alphapapa** [comment]
This is likely impractical due to the Vtable implementation which does not allow such customization.  Someone(TM) could implement something like that as another layer, but it's not something I have time for.

Please note that there is only one developer of this very new package.
